examples/pom.xml

```diff
@@ -84,7 +84,7 @@
     </dependency>
     <dependency>
       <groupId>io.dapr</groupId>
-      <artifactId>dapr-sdk-springboot</artifactId>
+      <artifactId>dapr-sdk-actors</artifactId>
```

Comment:
Is this correct is it supposed to be -examples?

Reply:
This is for dependency, not the name of this jar.